src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1084,10 +1084,12 @@ class DocBaseViewer extends BaseViewer {
             const canAnnotate = this.areNewAnnotationsEnabled() && this.hasAnnotationCreatePermission();
             const canDownload = checkPermission(this.options.file, PERMISSION_DOWNLOAD);
             const canHighlight = canAnnotate && canDownload;
+            const canDraw = canAnnotate && this.options.showAnnotationsDrawingCreate;
```

Comment:
Alphabetize.

Reply:
Thanks for catching, not sure how I missed this one.